README.md

```diff
@@ -1,1238 +1,21 @@
-# kubernetes
+# Kubernetes Ruby Client
```

Comment:
This is a generated file. You cannot edit it because it will be overwritten on next client generation. If you generate the client in a new folder, we won't have this problem as we will have a root level README.md file and a generated one in kubernetes/README.md.